cypress/e2e/permit2.test.ts

```diff
@@ -3,24 +3,17 @@ import { MaxUint160, MaxUint256 } from '@uniswap/permit2-sdk'
 import { DAI, USDC_MAINNET } from '../../src/constants/tokens'
 import { getTestSelector } from '../utils'
 
-const APPROVE_BUTTON = '[data-testid="swap-approve-button"]'
-
-/** Initiates a swap and confirms its success. */
-function swaps() {
+/** Initiates a swap. */
+function initiateSwap() {
   // The swap-button can be temporarily disabled following approval, & Cypress will retry clicking the disabled version.
   // This ensures that we don't click until the button is enabled.
   cy.get('#swap-button').should('not.have.attr', 'disabled')
 
```

Comment:
Actions will not run until their element is enabled: https://docs.cypress.io/guides/core-concepts/interacting-with-elements.

```suggestion
```